dist/stat.py

```diff
@@ -9,6 +9,24 @@
 from stat_data import groups, dsrc_stats, conn_stats, conn_dsrc_stats, join_stats, \
     session_stats
 
+##########################################
+# Check for duplicate stat descriptions:
+# Duplicate stat descpriptions within a category are not allowed.
+# The list must be sorted by description. 
+##################g########################
+def check_unique_description(sorted_list):
+    temp = ""
+    for i in sorted_list:
+        if temp == i.desc:
+            print("ERROR: repeated stat description in - '%s'" % (i.desc))
```

Comment:
Two things here: (1) is the `ERROR` enough to cause a patch build with invalid stats to fail? I'm a little surprised that we don't seem to need a `sys.exit(1)` or something.

(2) the grammar is a little bit off, the "in" makes it sound like the description is a category. I think we can just drop it.

Reply:
It is not the error code but rather an output from stat.py that gets error checked in s_all so the having an output is what causes the failure. 

In regards to the comment of "in", the captured description string is formatted with the category first and then the description. So it would look like "repeated stat description in - 'session: description here'" if that makes it better.

Reply:
OK, that looks better than what I had in my head - thanks for explaining!